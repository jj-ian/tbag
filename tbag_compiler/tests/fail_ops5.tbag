/* Authors: Maria */
room {}

room Closet { 
        name = "Closet";
}

room LivingRoom {
        name = "Living Room";
}

Closet <-> LivingRoom;

start { Closet }

currentRoom == "hello" {
        print("\n");
        endgame;
}
