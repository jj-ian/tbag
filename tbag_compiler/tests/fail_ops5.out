Fatal error: exception Failure("Types to equality operators ==, !=
                            must be the same and be integers, booleans, or
                            rooms")
